infra/bootstrap.sh

```diff
@@ -102,11 +102,17 @@ if [[ ! -z "${RUN_BOOTSTRAP:-}" ]]; then
     "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "automl-cpu-cluster" 0 4 "Standard_DS3_v2"
     # Larger CPU cluster for Dask and Spark examples
     "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "cpu-cluster-lg" 0 4 "Standard_DS15_v2"
+
+    echo_title "Ensuring CPU compute with MSI"
+    "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute_msi "cpu-cluster-msi" 0 20 "Standard_DS3_v2" "uaimevnet"
```

Comment:
Any reason of choosing 20 nodes for the CPU clusters?

Reply:
Since I am trying to replicate the cpu-cluster/gpu-cluster ..these clusters are getting used in all automl pipelines. 
So I choose the exact configuration that they have

Reply:
We will end up with outofquota error since we do not have unlimited quota for GPU in the subscription. Please reconfigure the quota numbers based on what these new clusters will be used for.